From: Robert Mader <robert.mader@posteo.de>
Date: Mon, 5 Oct 2020 13:16:40 +0200
Subject: clutter/stage-view: Make get_refresh_rate() public

We'll need it in a follow-up commit

Part-of: <https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/1468>

(cherry picked from commit f7cef11515f863c51383e9c6a7a5a1440c95e0fe)

Origin: upstream, 3.38.2, commit:a1db1dfa2e17b4c67c79960345b85b9c61a24546
---
 clutter/clutter/clutter-stage-view-private.h | 3 ---
 clutter/clutter/clutter-stage-view.h         | 3 +++
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/clutter/clutter/clutter-stage-view-private.h b/clutter/clutter/clutter-stage-view-private.h
index e413220..4bdb3d0 100644
--- a/clutter/clutter/clutter-stage-view-private.h
+++ b/clutter/clutter/clutter-stage-view-private.h
@@ -65,9 +65,6 @@ void clutter_stage_view_transform_rect_to_onscreen (ClutterStageView
 
 void clutter_stage_view_schedule_update (ClutterStageView *view);
 
-CLUTTER_EXPORT
-float clutter_stage_view_get_refresh_rate (ClutterStageView *view);
-
 void clutter_stage_view_notify_presented (ClutterStageView *view,
                                           ClutterFrameInfo *frame_info);
 
diff --git a/clutter/clutter/clutter-stage-view.h b/clutter/clutter/clutter-stage-view.h
index 4c54553..40c5afc 100644
--- a/clutter/clutter/clutter-stage-view.h
+++ b/clutter/clutter/clutter-stage-view.h
@@ -79,4 +79,7 @@ ClutterFrameClock * clutter_stage_view_get_frame_clock (ClutterStageView *view);
 CLUTTER_EXPORT
 CoglScanout * clutter_stage_view_peek_scanout (ClutterStageView *view);
 
+CLUTTER_EXPORT
+float clutter_stage_view_get_refresh_rate (ClutterStageView *view);
+
 #endif /* __CLUTTER_STAGE_VIEW_H__ */
